docs/troubleshoot/troubleshoot-zos.md

```diff
@@ -2,6 +2,16 @@
 
 The following topics contain information that can help you troubleshoot problems when you encounter unexpected behavior installing Zowe z/OS components or starting Zowe's `ZWESVSTC` started task.
 
+## Unable to launch Zowe with { FSUM7351 }
+
+When you run `zowe-start.sh` from a unix shell path `<zowe-instance-directory>/bin` you encounter the following error:
+
+```
+<RUNTIME_DIRECTORY>/scripts/internal/opercmd: ./zowe-start.sh 6: FSUM7351 not found
```

Comment:
This error can also occur if the permissions are incorrect I believe - ie if the runtime directory's ancestors don't have read/traverse permission.

Reply:
Good catch also.  I've updated the docs to reflect this